Sources/Rego/Builtins/Registry.swift

```diff
@@ -6,19 +6,24 @@ typealias Builtin = (BuiltinContext, [AST.RegoValue]) async throws -> AST.RegoVa
 struct BuiltinContext {
     public let location: OPA.Trace.Location
     public var tracer: OPA.Trace.QueryTracer?
+    /// Time of Context creation since epoch, in nanoseconds
+    public let timeNanos: UInt64
```

Comment:
⛏️ Should we store this natively as `Time` and only convert to nanos as needed?

Reply:
Yes, using `Date` instead in the next commit